website/pages/en/index.js

```diff
@@ -531,7 +454,13 @@ class Index extends React.Component {
                       non-Facebook contributors.
                     </translate>
                   </MarkdownBlock>
-                  <Contributors />
+                  <MarkdownBlock>
+                    <translate>
+                      We have over 50 Sponsors who contribute $100 or more per
+                      month, and over 450 Backers who have contributed $2 or
```

Comment:
I guess this was the intention?
```suggestion
                      We have over {sponsorCount} Sponsors who contribute $100 or more per
                      month, and over {backerCount} Backers who have contributed $2 or
```

Reply:
I couldn't get this working when I tried, probably should drop those counts - will give it one more shot

Reply:
Yeah, it doesn't handle interpolation, maybe I can wrap it into two

![Screen Shot 2020-08-15 at 10 05 42 AM](https://user-images.githubusercontent.com/49038/90314045-d3753000-dede-11ea-8ad4-68adadd3ed57.png)